### DISCUSSION TITLE
What is example.lua?

### DISCUSSION THREAD

**Discussion by naidaon** [LazyVim/LazyVim#2337] [Q&A] [answered]
Hi, I am brand new to neovim so this will be a newbie question. But when I install lazyvim I am curious if the file `lua/plugins/example.lua` is something that can be deleted? 

Kind of related tot he first question. What is the default theme of LazyVim and how can I change it?

**Comment by dpetka2001** [comment, marked as answer]
The `example.lua` is just a file that contains examples of different plugins' configurations. It doesn't actually do anything. You might want to read it through (it has some comments that might be helpful). You can also safely delete it, if you wish so. 

The default theme of LazyVim is `tokyonight` and to change it pls read here in the [docs](https://www.lazyvim.org/configuration#icons--colorscheme) where it's described what you have to do. The example there changes the colorscheme to `catppuccin`. You can change that to whatever colorscheme you would like.

**Reply by naidaon** [reply]
Thanks so much I really appreciate it :)
